tests/functional/modules/test_zos_encode_func.py

```diff
@@ -484,13 +487,17 @@ def test_uss_encoding_conversion_mvs_vsam_to_uss_file(ansible_zos_module):
             },
             backup=True,
         )
-        hosts.all.file(path=USS_DEST_FILE, state="absent")
         pprint(vars(results))
         for result in results.contacted.values():
             assert result.get("src") == MVS_VS
             assert result.get("dest") == USS_DEST_FILE
             assert result.get("backup_name") is not None
             assert result.get("changed") is True
+            # How can we add a content validation without having to encode again ?
```

Comment:
Some additional ideas for content validation could be that we use checksum(s) and/or fetch and diff if we don't want to do another encoding step. 

I think it would be good if we improved on the content validation as you note  for all these test cases rather than just checking the module state.

I'll create a new issue to improve the content validation and put it in the backlog.

Reply:
Nothing to resolve it was only a comment. See issue #410